stories/Image.stories.mdx

```diff
@@ -56,7 +56,7 @@ import { Image } from '@wpmedia/engine-theme-sdk';
             <p>Scroll down to see lazy loading example</p>
             <Image
                 url={'https://arc-anglerfish-arc2-prod-corecomponents.s3.amazonaws.com/public/37UMUNYNOVCEJDZW5SBKBXNMO4.jpg'}
-                alt={'This is a Free article for testing Mentor Medier’s paywall'}
+                alt={'Aerial view of bridge'}
```

Comment:
this was just the small alt tags that changed and caused storybook to think there had been a real code change